src/lib.rs

```diff
@@ -98,17 +98,36 @@ pub const DEFAULT_MAX_ALLOWED_MSG_SIZE: usize = 500 * 1024 * 1024; // 500MiB
 /// before using a random port.
 pub const DEFAULT_PORT_TO_TRY: u16 = 443;
 
+/// Senders for node and client events
+#[derive(Clone)]
+pub struct EventSenders {
+    /// The client event sender
+    pub client_tx: mpmc::Sender<Event>,
+    /// The node event sender
+    pub node_tx: mpmc::Sender<Event>,
```

Comment:
A general comment about naming these vars: I think it can be a bit confusing for the API user because `node_rx` / `client_rx` do not strictly relate to "nodes" only because I can use quic-p2p on the client side, and I may expect to receive e.g. a `BootstrappedTo` event - which in this case will be sent over the `node_tx` channel. So it's a bit counterintuitive that I'll need to use `node_rx` and _not_ `client_rx` for these events as a network client.

Probably a better name would be something like `outbound_tx` / `inbound_tx` or something like that?

Reply:
> do not strictly relate to "nodes" only because I can use quic-p2p on the client side, and I may expect to receive e.g. a BootstrappedTo event 

My understanding if we cannot bootstrap on a client.

Reply:
Updated docs.